torch/_export/__init__.py

```diff
@@ -792,15 +792,21 @@ def strip_root(x):
                 sig.buffers_to_mutate = pytree.tree_map(strip_root, sig.buffers_to_mutate)
                 return gm, sig
             return _aot_export_non_strict
-        ep_non_strict = _export_non_strict(f, args, {}, f.state_dict(), transform=_tuplify_outputs)
+
+        fake_mode, fake_args, dim_name_to_sources = make_fake_inputs(f, args, constraints)
+        ep_non_strict = _export_non_strict(f, fake_args, {}, f.state_dict(), transform=_tuplify_outputs)
+
+        range_constraints, equality_constraints = make_constraints(fake_mode, dim_name_to_sources, ep_non_strict.gm)
+
+
         assert out_spec is not None
         return ExportedProgram(
             ep_non_strict.gm,
             ep_non_strict.gm.graph,
             ep_non_strict.sig,
             _get_params_buffers(f),
-            {},
-            [],
+            range_constraints,
+            equality_constraints,
```

Comment:
not blocking but I feel maybe we could stop returning equality constraints as they will be removed and no one actively use them?

Reply:
I want to preserve behavior for now to minimize blast radius. I can zero out all equality constraints in a different PR (I was under the impression this was already done...)